packages/block/test/block.spec.ts

```diff
@@ -738,4 +738,8 @@ tape('[Block]: block functions', function (t) {
       st.end()
     }
   )
+
+  /*t.test('should set constant block fields if on the merge hardfork', function (st) {
+    //TODO
```

Comment:
@acolytec3 mentioned we already have `mergeBlock.spec.ts` added in this PR, @holgerd77 is there something more you were expecting to test here?

Reply:
oh I don't see the code that sets these fields anymore, just validation, maybe you mean adding that back. we can probably add these tests to `mergeBlock.spec.ts` then.

Reply:
nevermind the default values are already set to the merge defaults if not provided e.g.

```
difficulty ? new BN(toBuffer(difficulty)) : new BN(0),
...
extraData ? toBuffer(extraData) : Buffer.from([]),
mixHash ? toBuffer(mixHash) : zeros(32),
nonce ? toBuffer(nonce) : zeros(8),
```
etc.

Reply:
will remove for now so we can merge this in, let me know if i should do a follow-up PR if i misunderstood the todo intention here and if it's not covered by `mergeBlock.spec.ts:32`

Reply:
Not sure, Gabriel added this and I commented it out since it was breaking hte tests cause it had to `end()` call. I guess we are ok (for now) on the testing side I would say?